components/form-builder/translate/DownloadCSV.tsx

```diff
@@ -99,5 +99,9 @@ export const DownloadCSV = () => {
     URL.revokeObjectURL(url);
   };
 
-  return <FancyButton onClick={generateCSV}>Download .csv</FancyButton>;
+  return (
+    <Button onClick={generateCSV} theme="secondary">
+      Download .csv
```

Comment:
Maybe wrap this in a translation while your in here

Reply:
👍